docs/site/translation.md

```diff
@@ -33,38 +33,60 @@ Most of the following are relevant to locales at the Modern Coverage Level.
+- `punctuation-auxiliary` — If there are punctuation characters that are not customarily used, but may occur, add them here instead of in `auxiliary`.
+- `punctuation-person` — If there are punctuation characters that are customarily used in people's names in standard documents, add them here.
+This should be a small list such as “.” or “-”.
+Do **not** include ‘fanciful’ characters such as emoji or [kaomoji](https://en.wikipedia.org/wiki/Kaomoji).
 
 ### Locale display names
 
 #### More language names
 
-As new locales reach Basic Coverage, their language names have been added for locales targeting modern coverage: TBD
+As new locales reach Basic Coverage, their language names have been added for locales targeting modern coverage. These include: **TBD**
 
 #### Languages whose English name changed
 
-- tkl: English name changed to Tokelauan. [CLDR-11231](https://unicode-org.atlassian.net/browse/CLDR-11231)
+- tkl: English name changed to Tokelauan.
+<!-- Let's not add ticket numbers, except for Known Issues — there is already enough for vetters to read, and the tickets will often be confusing to people or take more time to read and puzzle out than is worth it. Fine to leave them in the document, but commented out, for our usage. 
+[CLDR-11231](https://unicode-org.atlassian.net/browse/CLDR-11231) -->
+
+#### Core/Extensions
+
+There is a new mechanism for better menu names. When you see a **Code** with `-core` or `-extension`, please read [Core and Extensions](/displaynames/languagelocale-names#core-and-extensions).
 
 #### Scripts
 
 There are 5 new scripts for Unicode 17. Currently the names are in English: Beria Erfe, Chisoi, Sidetic, Tai Yo, Tolong Siki.
-Coverage for other languages is at comprehensive; if there is a need to have coverage at lower level in some locale,
-please file a ticket. [CLDR-18283](https://unicode-org.atlassian.net/browse/CLDR-18283)
+Coverage for other languages is at comprehensive. 
+<!-- I don't think we want to encourage this: ... if there is a need to have coverage at lower level in some locale,
+please file a ticket. [CLDR-18283](https://unicode-org.atlassian.net/browse/CLDR-18283) -->
 
-#### ISO 8601 calendar name
+#### ISO 8601 calendar
 
 This is a variant of the Gregorian calendar whose formats always use year-month-day ordering and a 24-hour time cycle.
 The English name has changed to reflect that (and also added a variant); locales should update accordingly:
--calendar-iso8601: Gregorian (Year First)
--calendar-iso8601-variant: ISO 8601 Order
+- calendar-iso8601: Gregorian (Year First)
+- calendar-iso8601-variant: ISO 8601 Order
+
+Please go though the ISO8601 fields. You should change separators to match what is acceptable in your language. However, do not change the ordering of the elements, which should be strictly the following order (for any that occur in a particular pattern):
```

Comment:
I want to discuss this before we have vetters add data for this calendar type

Reply:
Ok. You can modify this PR for that (so people hold off on changes to iso8601 values), and we can discuss when I'm back next week. Sound good?